PowerEditor/src/WinControls/Preference/preference_rc.h

```diff
@@ -136,6 +136,11 @@
 
 	#define	IDC_CHECK_VIRTUALSPACE (IDD_PREFERENCE_SUB_EDITING + 45)
 
+	#define	IDC_CARETLINEFRAME_WIDTH_CHECK (IDD_PREFERENCE_SUB_EDITING + 46)
+	#define	IDC_CARETLINEFRAME_WIDTH_STATIC (IDD_PREFERENCE_SUB_EDITING + 47)
+	#define	IDC_CARETLINEFRAME_WIDTH_SLIDER (IDD_PREFERENCE_SUB_EDITING + 48)
+	#define	IDC_CARETLINEFRAME_WIDTH_DISPLAY (IDD_PREFERENCE_SUB_EDITING + 49)
+

```

Comment:
Please erase your local `preference_rc.h` by using this one.

[preference_rc.zip](https://github.com/notepad-plus-plus/notepad-plus-plus/files/8483401/preference_rc.zip)